### DISCUSSION TITLE
unable to import custom cursor from local path

### DISCUSSION THREAD

**Discussion by auditism** [tailwindlabs/tailwindcss#9452] [Help] [answered]
I would like to add a custom cursor, when I input the url like so, everything works as expected;

```
// tailwindconfig.js
module.exports = {
    content: ["./pages/**/*.{js,ts,jsx,tsx}", "./components/**/*.{js,ts,jsx,tsx}"],
    theme: {
        extend: {
            cursor: {
                default: "url(https://play.tailwindcss.com/favicons/favicon-16x16.png?v=3), default",
            },
        },
    },
    plugins: [],
}
```
But if I put the path of a local file it won't work, this way;

```
// tailwindconfig.js
module.exports = {
    content: ["./pages/**/*.{js,ts,jsx,tsx}", "./components/**/*.{js,ts,jsx,tsx}"],
    theme: {
        extend: {

            cursor: {

                default: "url(/constants/mouse-pointer/favicon-16x16.png), default",
            },
        },
    },
    plugins: [],
}
```
There are no errors, the cursor will just not not change..

I am using Next.js and tailwind CSS, and I've been able to make it work with CSS alone, do you guys know what I might be doing wrong?

**Comment by wongjn** [comment, marked as answer]
It seems like the URL may not be resolving to an image file – check the URL that it is compiled to in devtools and see if you can navigate to it with a non 404 response. For example, if the `url()` value you supplied goes through your build pipeline as-is, check that you can access the image file at https://your.domain/constants/mouse-pointer/favicon-16x16.png.

**Comment by auditism** [comment]
Yes you are right ! I am indeed getting a 404. Thank you
